File: bulk/csharp/HardcodedCreds01.cs
// CWE-798: Hardcoded connection string
using System.Data.SqlClient;
public class HardcodedCreds01 {
    private const string ConnStr = "Server=db;Database=app;User=sa;Password=P@ssw0rd!";
    public SqlConnection GetConnection() => new SqlConnection(ConnStr);
}

File: bulk/java/HardcodedCreds01.java
// CWE-798: Hardcoded database credentials
import java.sql.*;
public class HardcodedCreds01 {
    private static final String DB_PASS = "Sup3rS3cr3t!";
    public Connection getConn() throws SQLException {
        return DriverManager.getConnection("jdbc:mysql://db:3306/app", "root", DB_PASS);
    }
}
